packages/flame/lib/src/components/spawn_component.dart

```diff
@@ -14,7 +14,7 @@ import 'package:flame/math.dart';
 /// components.
 /// If you want to use a non static time interval, use the
 /// [SpawnComponent.periodRange] constructor.
-/// {@endremplate}
+/// {@endtemplate}
```

Comment:
note how a more aggressive regex can catch actual bugs and save lives! (well at least the former)